core/remote/watcher/index.js

```diff
@@ -112,7 +112,7 @@ class RemoteWatcher {
 
       try {
         let target = -1
-        errors = errors.concat(await this.pullMany(docs))
+        errors = await this.pullMany(docs)
```

Comment:
You are sure about that change? I think that `errors` should be an array

Reply:
`pullMany` is still returning an array but since we don't have any `errors` prior to this call I figured we didn't need to concatenate the result.